### ISSUE TITLE
Arrows with Global Traveler Trait in Off-Hand

### ISSUE THREAD

**Issue by JHawkley** [Landmaster/PlusTiC#292]
_Using PlusTiC as a part of the Enigmatica 2 Expert mod-pack, version 1.64d._

I crafted some arrows with an osglogas arrow head which grants it the Global Traveler trait.  I successfully linked them to an inventory (a Small Storage Crate from Actually Additions, but I also tried a vanilla MineCraft chest) while the arrows were in my main-hand.

When the arrows are held in the main-hand with the bow in my off-hand, items from a defeated mob are transported to the target inventory correctly.

If the arrows are in my off-hand (which is generally where I like to put them) and the bow in my main-hand, the items from a mob spawn on the ground when defeated using the arrows.

I expected the Global Traveler trait to activate whenever the arrow is fired and used to defeat a mob, regardless of which hand it is held in or even if its just being used from the player's inventory.

Since I like to use my bow as a melee attack, having my bow in my off-hand ends up being a bit inconvenient, since its melee attack cannot be used in that configuration.

Also, tangentially related, it'd be nice if the Brown Magic trait's shortcut keys also worked while a tool with that trait was held in the off-hand.  😛

**Comment by Landmaster** [comment]
> Also, tangentially related, it'd be nice if the Brown Magic trait's shortcut keys also worked while a tool with that trait was held in the off-hand. 😛

The problem with that is handling the case where *both* tools have the trait…

**Comment by JHawkley** [comment]
I don't think it would be too bad to just give priority to the tool in the main-hand, in that case.
